### PR TITLE
Create migration

### PR DESCRIPTION
**Pull request by QuanLuu97**
ready

### CODE REVIEW

**Comment by QuanLuu97** [comment]
ready

File: project1/database/migrations/2018_08_02_025355_create_companies_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('companies', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('phone');
+            $table->string('email');
+            $table->string('addredd');
```
Comment (BunnyPi04) [inline project1/database/migrations/2018_08_02_025355_create_companies_table.php]: sai chính tả

**Comment by BunnyPi04** [comment]
commented

**Comment by QuanLuu97** [comment]
ready

**Pull request merged**